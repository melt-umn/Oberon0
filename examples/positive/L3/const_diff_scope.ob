MODULE ConstDiffScope;

CONST x = 42;

PROCEDURE f;
  CONST x = 40;
  BEGIN
    Write(x); WriteLn;
  END f;
BEGIN
  f;
  Write(x); WriteLn;
  f;
END ConstDiffScope.
