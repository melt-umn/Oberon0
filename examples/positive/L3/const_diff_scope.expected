 40
 42
 40
